README.md

```diff
@@ -716,7 +717,7 @@ case?  R's **nrow** function tells us the number of rows in any data
 frame:
 
 ``` r
-> nrow(ToothGrowth)
+> nrow(tg)
```

Comment:
Previous and following code blocks use **tg** as shorthand for **ToothGrow** so these can be changed to **tg** to stay consistent